### DISCUSSION TITLE
Show off your awesome websites!

### DISCUSSION THREAD

**Discussion by jaredcwhite** [bridgetownrb/bridgetown#201] [Show and tell]
We've seen a bunch of new Bridgetown sites pop up lately! Here's your official thread to post links to cool stuff you've been working on. 😎

**Comment by jaredcwhite** [comment]
Just came across this site by @andypeters — looking very nice! https://andypeters.com

**Reply by andypeters** [reply]
Thank you for the kind words @jaredcwhite.  😎  

I left the repo public if anyone is interested:  https://github.com/andypeters/andypeters.com

**Comment by jaredcwhite** [comment]
This is a really neat marketing site by @geetfun for his new SaaS help desk, Eager: https://eager.app

**Reply by ghost** [reply]
Is there any ready-to-use template library for bridgetownRB, like Jekyll themes?

**Reply by monfresh** [reply]
Not that I have found, but I'm working on something similar as part of the Ultimate version of [Ruby on Mac](https://www.rubyonmac.dev/#ultimate). The plan is to create a script that will automatically generate a new Bridgetown site ready to go with Tailwind, SEO, analytics, syntax highlighting, and ready to be deployed to GitHub Pages, Netlify, and other hosting services.

**Comment by bugthing** [comment]
Hi @jaredcwhite :wave:  thanks so much for making BridgetownRB. Loving your work with the project and it gets better and better.
I converted a wordpress site I maintain for a family members charity: http://www.willhoughtonfoundation.org/
The site was really slow :snail:  but now it whips along in comparison :speedboat:
There is still some work/polish to do but I am pleased with the result.
Thanks again :medal_sports:

**Reply by jaredcwhite** [reply]
Looks awesome @bugthing! Pretty snappy over here in the U.S. :+1:

**Comment by luctus** [comment]
Hey @jaredcwhite we just deployed the first version of our discounts marketplace using bridgetown! Thanks a lot for all the help you and the team (@ayushn21) gave me during the process. It was super fun and I learnt a lot!

As a context, we are offering group verification to e-commerce websites and for that we built a Ruby on Rails app, our clients were including our "widget" (rendered with Rails) in their websites. Now we built a website to list all the current e-commerce sites that we are working with.

You can check the final result here: https://shop.gocertify.me

It's built with bridgetown-v1.0.0.alpha11 + tailwind and deployed to Netlify.

Cheers!

**Reply by jaredcwhite** [reply]
Good to hear @luctus – looks awesome!

**Comment by rhoopman** [comment]
I'm currently working on a project for a small dutch boat rental company.

For now it's a small fleet of 26ft boats, retrofitted with fully electrical propulsion. 

Built with bridgetown 0.21.1/tailwind/alpinejs and deployed to render.com. A bit of a learning curve coming, but very much worth it.

[de Fluistervloot](https://defluistervloot.nl)

**Comment by monfresh** [comment]
Hey @jaredcwhite. Thanks for building Bridgetown. I used it to create [Ruby on Mac](https://www.rubyonmac.dev/?utm_source=github&utm_campaign=bridgetown-discussions): a script that sets up a working Ruby environment with a single command.

I used Bridgetown 1.0.0beta1 + Tailwind UI and deployed it to Netlify.

**Comment by tommasongr** [comment]
Here a bunch of websites I made with Bridgetown. Thanks to @jaredcwhite and everyone involved in this amazing project and community!

- https://tommasonegri.com/ (personal site, redesign coming up soon)
- https://www.scarioni.org/ (precision mechanics company site)
- https://www.escavamar.com/ (premium marbles company site)
- https://www.morphzine.com/ (editorial project site)

I'm currently working on another three Bridgetown based projects and I hope to grow the list even more in the future 🚀
